Vindicta.Altis/config/global_config.hpp

```diff
@@ -1,48 +1,15 @@
+// DEFAULT DEBUG CONFIG
+// If you want to modify for yourself only then use user_local_config.hpp in this same directory (it will not be committed).
 
-// Default oop config overrides.
-// Nothing here cos we don't want to override any stuff by default!
-// Undefine debug and info logging, leave warning and error logging.
-//#undef OOP_DEBUG
-//#undef OOP_INFO
-//#undef OOP_WARNING
-//#undef OOP_ERROR
+// Various runtime OOP assertions (class existence, member variable existence, etc)
+#define OOP_ASSERT
```

Comment:
does this add checks and lower perf ? maybe we should define it to undef by default to ease release build ?

Reply:
This applies only to debug build, release uses `config/release/global_config.hpp`. Although looking now we have 3 global_config.hpp, configs\dev configs\release and Vindicta.Altis. Not sure anymore which are used when.

Reply:
Actually I guess I remember now, it uses the Vindicta.Altis one, and copies over it for making the pbos. So this change is okay I guess? We want asserts on or off by default when played through editor? Maybe off?

Reply:
hum ye ok i see well then maybe it make sens to define more debug thing by default if there is a release_config file